moderatorguide.md

```diff
@@ -1,30 +1,32 @@
 # Moderator Guide
 
-## Summary
+## Your tasks
+ 
 
-Moderation of a virtual conference involves more responsibility than chairing an offline session.
-Speaker-audience interaction is important for the success of a talk, but in a virtual environment it becomes more complicated.
-The moderator takes on the following responsibilities: 
-
-* announce and introduce the conference,
-* remind the audience of the seminar format, when questions are allowed, and how to raise hands to ask questions,
-* initiate a seminar by highlighting the speaker’s screen-share, unmuting their microphone, and introducing them by announcing their name, 	institution, and talk title,
+* open the program by introducing the event, organizers and the (first) speaker,
+* remind the audience of the event format and explain how and when to ask questions
 * monitor the participant list for raised hands and enable those attendees to ask questions,
-* close a seminar by thanking the speaker for their talk, muting and un-highlighting the speaker, and making any transition announcements,
 * handle any technical issues with the execution of the virtual conference,
-* monitor participant behavior in the chat for breaches of the code of conduct.
+* monitor participant behavior in the chat for breaches of the code of conduct,
+* give a closing remarks.
 
 Below we provide more detailed information on certain aspects of the moderator role. 
 
 ## How to
 
-### Transition seminars
+### Opening and closing remarks
+
+At the beginning of the seminar: 
 
-A seminar is ready to start, highlight the speaker’s screen-share, introduce the speaker (name, current institution, something about the talk or their current work), and then unmute the speaker to allow them to begin their seminar.
+* highlight the speaker’s screen-share,
+* introduce the speaker (name, current institution, something about the talk or their current work),
+* unmute the speaker and invite them to begin their seminar.
 
-When a seminar has concluded, thank the speaker and then enter the post-talk question/answer period if there will be one (see guidelines on questions below).
-After this, mute their microphone, and stop highlighting their screen-share.
-If the seminar ended early then you may pause until it is time for the next one to start.
+At the end of the seminar: 
+
+* thank the speaker 
+* initiate the post-talk question/answer period
+* stop highlighting speaker's screen-share.
```

Comment:
Similar to above: the action is called "spotlighting", and it applies to the webcam stream and not the screen share.